### ISSUE TITLE
✨ Figure out static auth-less pages

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#7809]
> __Warning__
This issue is a bit vague at the moment.  It needs to be fleshed out.

## ✨ Feature

There are a few scenarios when it would be nice to have a static landing page that doesn't reboot the entire app.  A lot times, you want the user to be at these pages specifically because 

- When returning from SiC logout the app reboots three times:
   - initial boot of /logged-out
   - redirected boot of /en/logged-out
   - auth reboots the app one more time "just to be sure"
   
- When you try to log in as a deleted user   
  - You get logged out (see above)
  - The app reissues all the "me" context queries
  - You end up on the "deleted user"
  
 - A real 404 page that doesn't have to boot the entire app

We do have the absolute-bare-bones "secure network only" static page, too.

### :question: Questions

- Can we make something that looks like the rest of the app without much trouble?
- Some kind of static site generator that can read React code?

## 🎨 Design File

Include a link to the design file (if it exists).

### 🧑‍🎨 Designer

Include the author of the linked design file's name (or even better, their `@githubUsername`) for future reference.

## 📸 Screenshot

Add a screenshot of the new feature (if possible).

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Criteria 1
- [ ] Criteria 2

**Comment by esizer** [comment]
Just some thoughts since we were discussing this yesterday and I have had some time to sit and think about this. 

First, I'm going to do a small nudge by saying, I feel like a lot of this could be smoothed out if we finally adopt SSR where we can do a lot of this on the server before we even boot the app in the client for the user 😅 This would help with a lot of the jank associated with rebooting the app multiple times to get all of the data we need. 😉 

In the meantime, I think we could spin up a super light weight site with [Astro](https://astro.build/). The benefit here would be, we could have an entirely static site to send users when we don't need a lot of interactivity or when they logout. The big benefit with Astro is that it allows React (or any of the popular frontend frameworks) and has the concept of [islands](https://docs.astro.build/en/concepts/islands/). 

We could use our existing components and, using islands, we could have parts of those pages hydrated to show dynamic content if needed. Otherwise, the rest of the site would be just HTML + CSS. We only need to load the JavaScript required by dynamic portions like loading the currently logged in user in the header.

**Comment by tristan-orourke** [comment]
This seems like it overlaps with #7866 and #7867. Maybe let's turn this one into a discussion?

**Comment by petertgiles** [comment]
Good call.  Those two issues are a little more focused after our discussion in retro yesterday.  I'll convert this one.
